net-im/libcommuni/libcommuni-3.5.0_p20180116.ebuild

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+MY_PV="d3e388de9a146faad3277b46e480b0f1415f9a24"
+
+inherit qmake-utils
+
+DESCRIPTION="A cross-platform IRC framework written with Qt"
+HOMEPAGE="http://communi.github.io/"
+SRC_URI="https://github.com/communi/${PN}/archive/${MY_PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="qml test +uchardet"
+
+# Compiling with USE="test" is currently broken with dev-libs/icu
+# Compiling with USE="test" fails on one test
```

Comment:
The one test fails new in this version bump. Should I patch it out?

Reply:
Ok, I've disable the specific test and reenabled the tests, since this version do run find the tests with `dev-libs/icu`. Locally, tests now pass for me with `dev-libs/icu` and `dev-libs/uchardet`.